packages/core/src/framebuffer/Framebuffer.js

```diff
@@ -55,11 +55,11 @@ export default class Framebuffer
     addColorTexture(index = 0, texture)
     {
         // TODO add some validation to the texture - same width / height etc?
-        this.colorTextures[index] = texture || new Texture(null, { scaleMode: 0,
+        this.colorTextures[index] = texture || new BaseTexture(null, { scaleMode: 0,
```

Comment:
Previous it was `import Texture from '../textures/BaseTexture';` which is a little confusing since Texture is not the same as BaseTexture.

I can change this to `import { BaseTexture as Texture } from '../textures/BaseTexture'` if you prefer